### PR TITLE
Upgrade to RN 0.64.1, upgrade some dependencies, allow full screen videos

### PR DESCRIPTION
**Pull request by pmusaraj**
- Includes dependabot updates
- Switches to using Hermes in iOS (new feature in React Native, faster app boot)
- Update launch screen background so it matches OS mode (light or dark)
- Bumps app version to 1.7.0 

Build for this version is now ready for testing on TestFlight and on the Play Store (alpha version).

### CODE REVIEW

File: ios/Discourse.xcodeproj/xcshareddata/xcschemes/Discourse-tvOS.xcscheme [hunk 1]
```diff
@@ -1,88 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (CvX) [inline ios/Discourse.xcodeproj/xcshareddata/xcschemes/Discourse-tvOS.xcscheme:1]: Does this mean Discourse is no longer available on tvOS? 😅 (I don't know if it really was before 😛)

File: fastlane/Fastfile [hunk 2]
```diff
@@ -2,8 +2,8 @@ fastlane_require "base64"
 fastlane_require "fileutils"
 fastlane_require "json"
 
-IOS_APP_VERSION = "1.6.9"
-ANDROID_APP_VERSION = "1.6.5"
+IOS_APP_VERSION = "1.7.0"
+ANDROID_APP_VERSION = "1.6.6"
```
Comment (CvX) [inline fastlane/Fastfile:6]: Any reason we don't want these version numbers synchronized?

**Review by CvX — APPROVED** [review]
Good stuff!

File: ios/Discourse.xcodeproj/xcshareddata/xcschemes/Discourse-tvOS.xcscheme (hunk 1 above)
Comment (pmusaraj) [reply ios/Discourse.xcodeproj/xcshareddata/xcschemes/Discourse-tvOS.xcscheme:1]: Naah, we never tested/released the app for tvOS.

File: fastlane/Fastfile (hunk 2 above)
Comment (pmusaraj) [reply fastlane/Fastfile:6]: Not a strong reason, but the Android version is a lot simpler, given that it doesn't use the webview for the sites. So, the discrepancy is due to having more iterations for iOS with fixes and tweaks to the webview.

**Pull request merged**